test/e2e/workflow_test.go

```diff
@@ -224,7 +224,7 @@ var _ = Describe("Kogito Serverless Operator", Ordered, func() {
 
 			EventuallyWithOffset(1, func() error {
 				cmd := exec.Command("kubectl", "delete", "-f", filepath.Join(projectDir,
-					"config/samples/"+test.KogitoServerlessWorkflowSampleDevModeYamlCR), "-n", namespace)
+					"test/samples/"+test.KogitoServerlessWorkflowSampleDevModeYamlCR), "-n", namespace)
```

Comment:
Can u do the same here instead of using consts?

Reply:
Ok, moved the constants here